spec/fuzzy-finder-spec.coffee

```diff
@@ -585,6 +585,20 @@ describe 'FuzzyFinder', ->
           expect(atom.workspace.panelForItem(projectView).isVisible()).toBe false
           expect(inputView).toHaveFocus()
 
+  describe "when the filter text contains an absolute path", ->
```

Comment:
it's not really relative.... it's "normalized" , you can't do like ../current_file.html
although it could be interesting

but you cant do  
 ~/.atom/config.cson or /home/me/../hello